packages/google_sign_in/google_sign_in_web/lib/src/utils.dart

```diff
@@ -72,8 +72,8 @@ GoogleSignInUserData? gisResponsesToUserData(
   return GoogleSignInUserData(
     email: payload['email']! as String,
     id: payload['sub']! as String,
-    displayName: payload['name']! as String,
-    photoUrl: payload['picture']! as String,
+    displayName: payload['name'] as String?,
+    photoUrl: payload['picture'] as String?,
```

Comment:
This makes total sense. Even the documentation of `GoogleSignInUserData` has the following comment on top of `displayName` and `photoUrl`:

```
/// Not guaranteed to be present for all users, even when configured.
```